cypress/e2e/use-cases/migration.spec.js

```diff
@@ -45,6 +47,8 @@ describe('Saved requests in the legacy database should be correctly migrated', (
     });
 
     it('Migrates a normal proceeding', () => {
+        skipOn(isOn('production'));
```

Comment:
Why do we skip these?

Reply:
The APIs for accessing the stores are only available in dev.